### PR TITLE
Add more about PQLs and onboarding

### PR DESCRIPTION
**Pull request by malomarrec**
Feel free to merge upon review

### CODE REVIEW

**Comment by malomarrec** [comment]
@ebrodymoore feel free to merge upon review

File: content/departments/engineering/teams/growth/cloud-trial-operations.md [hunk 1]
```diff
@@ -63,11 +63,20 @@ Note: all automated email will automatically appear in salesforce (through Malo'
...
+- the inbound SDR that owns the lead will create a #cloud-trial-companyname channel, if it doesn't exist yet, and add the AE, CE, Greg Bastis, Nick Gage, Andrew Reed, Eric Brody-Moore and Malo Marrec.
+
 ### PQL qualification workflow
 
 The product growth team will be monitoring analytics for trial instances that have usage patterns that show signs that they could convert into a customer: Product Qualified Leads (PQL). For now, the criteria is best judgement and the workflow is manual, but this will be clearly defined and automated in the future.
 
-When a trial becomes a PQL, the lead status will be changed to `PQL` in Salesforce, which will trigger an alert in #product-qualified-leads, to be picked up by an Account Executive.
+When a trial becomes a PQL:
+
+- the lead status will be changed to `PQL` in Salesforce, which will trigger an alert in #product-qualified-leads, to be picked up by an Account Executive.
```
Comment (ebrodymoore) [inline content/departments/engineering/teams/growth/cloud-trial-operations.md]: is this workflow done?

Comment (malomarrec) [reply content/departments/engineering/teams/growth/cloud-trial-operations.md]: No

Comment (malomarrec) [reply content/departments/engineering/teams/growth/cloud-trial-operations.md]: Now yes

**Pull request merged**